package.json

```diff
@@ -17,19 +17,16 @@
     "lint:fix": "eslint . --fix && turbo run lint:fix",
     "test": "turbo run test --parallel",
     "test:watch": "turbo run test --watch",
-    "clean": "rm -rf node_modules && rm -rf */*/node_modules",
+    "clean": "rm -rf node_modules && turbo run clean",
```

Comment:
This mean all package.json need to have a `clean` command (would need to add in handlebars files). Also if we remove node_modules we wont have turbo as dependecy :) so the command will fail.

Reply:
Ahh! Good catch I had turbo installed globally maybe that's the reason everything worked. Have change the command to:
```
"clean": "turbo run clean && rm -rf node_modules",
```